madmom/features/beats.py

```diff
@@ -48,38 +49,47 @@ class RNNBeatProcessor(SequentialProcessor):
 
     """
 
-    def __init__(self, post_processor=average_predictions, **kwargs):
+    def __init__(self, post_processor=average_predictions, online=False,
+                 nn_files=None, **kwargs):
```

Comment:
I understand. But then this use case should be explained in the docs, I think it's not obvious from the beginning. Now that I've checked, the `online` parameter is also missing from the docstring of the class.